### PR TITLE
fix: update en-us dict

### PR DESCRIPTION
**Pull request by arkid15r**
# Fix Dictionary

Dictionary: en-us

## Description

Add a few new words

## References

N/A

## Checklist

- [x] By submitting this pull-request, you agree to follow our [Code of Conduct](https://github.com/streetsidesoftware/cspell-dicts/blob/main/CODE_OF_CONDUCT.md)
- [x] Verify that the title starts with the correct prefix:
  - `fix:` - for minor changes like adding words or fixing spelling issues.
  - `feat:` - for a significant change like adding a whole new set of words to a dictionary.
  - `feat!:` - for breaking changes, like file format or licensing changes.
  - `chore:` - for changes that do not impact the content of dictionaries.

### CODE REVIEW

File: dictionaries/en_US/src/en_US.txt [hunk 1]
```diff
@@ -1960,6 +1960,7 @@ blasty
 blatter
 blattering
 blatters
+blazingly
```
Comment (Jason3S) [inline dictionaries/en_US/src/en_US.txt:1963]: Please add new words to `src/additional_words.txt` instead of `en_US.txt`. `en_US.txt` is the original word list.

File: dictionaries/en_US/src/acronyms.txt [hunk 2]
```diff
@@ -16,5 +16,7 @@ scotus
 SCOTUS
 tldr
 TLDR
+usd
+USD
```
Comment (Jason3S) [inline dictionaries/en_US/src/acronyms.txt:20]: Wow, I had no idea USD was missing from the dictionary.

File: dictionaries/en_US/src/en_US.txt [hunk 3]
```diff
@@ -22712,6 +22715,7 @@ whists
 whitecapper
 whitecapping
 whitecomb
+whitepsace
```
Comment (Jason3S) [inline dictionaries/en_US/src/en_US.txt:22718]: `whitespace` is already in the US English dictionary.

The command `trace` command can be used to find words. `cspell trace whitespace`

image

Comment (arkid15r) [reply dictionaries/en_US/src/en_US.txt]: Yep, found it this way. Thanks for the `trace` 👍

File: dictionaries/en_US/src/en_US.txt (hunk 1 above)
Comment (arkid15r) [reply dictionaries/en_US/src/en_US.txt]: Moved them to the right location

File: dictionaries/en_US/src/acronyms.txt (hunk 2 above)
Comment (arkid15r) [reply dictionaries/en_US/src/acronyms.txt:20]: I know, that was unexpected.

**Review by arkid15r — COMMENTED** [review]
Fixed the comments, please review again when you get a chance.

Thank you!
